ffbs-mesh-vpn-parker/files/usr/sbin/nodeconfig.sh

```diff
@@ -30,6 +31,10 @@ fi
 
 pubkey=$(sed 's/+/%2B/g' /etc/parker/wg-pubkey)
 
+if ! { echo "untrusted comment: signify public key" && uci -q get parker.nodeconfig.config_pubkey; } > "${SIGN_PUB_KEY}"; then
+    $LOGGER unable to read or write pubkey from uci
```

Comment:
Is there any use in continuing this script if this fails? Without this key we won't be able to download a config anyway.

Reply:
I wasn't sure about the side-effects of exiting the script here and it will fail anyway if there is no PUBKEY later on in line 69 (`usign -V ...`).

I am happy to `exit 1` early here if you think it's OK.

Reply:
Yeah, that should be ok. The effect will be the same: Will never download a config nor get our Wireguard tunnels configured. So a node would just stay *client* in this case.

Reply:
Added the exit1